src/components/hds-copy-snippet/copy-snippet.module.css

```diff
@@ -111,7 +111,6 @@
 
 .hds-copy-snippet__text {
 	flex: 1 0 0;
-	max-width: calc(100% - 24px);
```

Comment:
Resolves wrapping edge case in [✓ copy snippet wrapping oddly (see screenshot)](https://app.asana.com/0/0/1205577920166090/f)